src/tasks/vision/face_landmark/builder.rs

```diff
@@ -0,0 +1,107 @@
+use super::{FaceDetectorBuilder, FaceLandmarker, TensorType};
+
+use crate::model::ZipFiles;
+use crate::tasks::common::{BaseTaskOptions, FaceLandmarkOptions};
+
+/// Configure the build options of a new **Face Landmark** task instance.
+///
+/// Methods can be chained on it in order to configure it.
+pub struct FaceLandmarkerBuilder {
+    pub(in super::super) base_task_options: BaseTaskOptions,
+    pub(in super::super) face_landmark_options: FaceLandmarkOptions,
+}
+
+impl Default for FaceLandmarkerBuilder {
+    #[inline(always)]
+    fn default() -> Self {
+        Self {
+            base_task_options: Default::default(),
+            face_landmark_options: Default::default(),
+        }
+    }
+}
+
+impl FaceLandmarkerBuilder {
+    /// Create a new builder with default options.
+    #[inline(always)]
+    pub fn new() -> Self {
+        Self {
+            base_task_options: Default::default(),
+            face_landmark_options: Default::default(),
+        }
+    }
+
+    base_task_options_impl!(FaceLandmarker);
+
+    face_landmark_options_impl!();
+
+    pub const FACE_DETECTOR_CANDIDATE_NAMES: &'static [&'static str] = &["face_detector.tflite"];
+    pub const FACE_LANDMARKS_CANDIDATE_NAMES: &'static [&'static str] =
+        &["face_landmarks_detector.tflite"];
+
+    /// Use the current build options and use the buffer as model data to create a new task instance.
+    #[inline]
+    pub fn build_from_buffer(
+        self,
+        buffer: impl AsRef<[u8]>,
+    ) -> Result<FaceLandmarker, crate::Error> {
+        face_landmark_options_check!(self);
+        let buf = buffer.as_ref();
+
+        let zip_file = ZipFiles::new(buf)?;
+        let face_detection_file = search_file_in_zip!(
+            zip_file,
+            buf,
+            Self::FACE_DETECTOR_CANDIDATE_NAMES,
+            "FaceDetection"
+        );
+        let landmark_file = search_file_in_zip!(
+            zip_file,
+            buf,
+            Self::FACE_LANDMARKS_CANDIDATE_NAMES,
+            "FaceLandmark"
+        );
+
+        let subtask_face_detector = FaceDetectorBuilder::new()
+            .device(self.base_task_options.device)
+            .num_faces(self.face_landmark_options.num_faces)
+            .min_detection_confidence(self.face_landmark_options.min_face_detection_confidence)
+            .build_from_buffer(face_detection_file)?;
+
+        // parse model and get model resources.
+        let model_resource = crate::model::parse_model(landmark_file.as_ref())?;
+
+        // check model
+        model_base_check_impl!(model_resource, 1, 2);
```

Comment:
I used `mediapipe-tasks` because it shares the same download path as the other models, but I agree that switching to `mediapipe-models` is a good idea since it's more up-to-date. This will require a few changes, so I think it makes sense to do it in a new PR, where I'll also add the blendshapes handling.

Reply:
I agree. Let's have another PR for this.